### ISSUE TITLE
WCAG2ICT Project: Add from WCAG 2.2 - 3.3.8 Accessible Authentication (Minimum)

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#42]
- [x] Add [3.3.8 Accessible Authentication (Minimum)](https://www.w3.org/TR/WCAG22/#accessible-authentication-minimum) section (from WCAG 2.2)
- [x] Add interpretation of term "cognitive function test", if needed.
- [x] Add in the [Understanding 3.3.8 Accessible Authentication (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/accessible-authentication-minimum.html) Intent and Benefits sections
- [x] Consider and write up whether there are any notes needed for application of this SC to non-web technologies

**Comment by maryjom** [comment]
## Cognitive function test

A task that requires the user to remember, manipulate, or transcribe information. Examples include, but are not limited to:

- memorization, such as remembering a username, password, set of characters, images, or patterns. The common identifiers name, e-mail, and phone number are not considered cognitive function tests as they are personal to the user and consistent across websites;
- transcription, such as typing in characters;
- use of correct spelling;
- performance of calculations;
- solving of puzzles.

This definition seems to be able to apply as defined, without any substitution, modification, or notes.

**Comment by maryjom** [comment]
NOTE: Latest work in AG WG may be  adding "(Minimum)" to the short name so it reads "Accessible Authentication (Minimum)".

**Comment by maryjom** [comment]
@ferBonnin I think we can start working on WCAG 2.2, as final publication is imminent.

**Comment by Lboniello** [comment]
I’ve not but can definitely make a dent this weekend. Feel free to send your thoughts and I’ll try to incorporate. If you prefer a conversation (I know Gregg had an opinion), please send some times that work for you on Monday. LauraSent from my iPhoneOn Dec 7, 2023, at 8:16 AM, Phil Day ***@***.***> wrote:﻿
@Lboniello & @GreggVan I forgot about this one this week. Have either of you made any progress?

**Comment by maryjom** [comment]
@Lboniello @pday1 @GreggVan Please be prepared to discuss progress/plans for addressing this note so we can get the guidance for this SC approved by the TF.

**Comment by pday1** [comment]
Proposed note for discussion:

Note Systems that are designed for shared use (such as in a public library) are out of scope for this requirement, as the user will not have access to systems that allow them to auto-complete content. 

Note Some closed systems, particularly those that handle financial transactions, have a requirement for a personal identification number (PIN) for authentication. This is a security requirement set down by the banking industry, and currently there is no globally acceptable alternative to the PIN for some systems (e.g. card payment terminals and ATMs). These systems are therefore out of scope for this requirement as users must use their PIN (either the digits themselves, or the spatial location of the digits on the keypad) in order to meet the security requirements. 

--
@GreggVan @Lboniello Please review and add your comments & edits.

**Comment by maryjom** [comment]
Guidance for 3.3.8 was added via PR #305.

**Comment by nitedog** [comment]
Another proposed note: on first start of the product, login is required to start the installation process (using your profile)

**Comment by maryjom** [comment]
@nitedog I'm not exactly sure where you suggest this note be added. Can you please create a draft PR or use this [google doc for 3.3.8](https://docs.google.com/document/d/18giKt9bddNEgnVmn3f8esr5SGhzJlf6vvX8MyBUmK48/edit?usp=sharing) to provide the exact proposal. I'm trying to create the final content updates being proposed into surveys I send out today.

**Comment by maryjom** [comment]
The TF surveyed the proposed changes (see [survey results](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-public-comments-R4/results#xq7)) and discussed in last week's meeting ([see 16 May minutes](https://www.w3.org/2024/05/16-wcag2ict-minutes.html#t11)). A [resolution on 3.3.8](https://www.w3.org/2024/05/16-wcag2ict-minutes.html#r08) was reached to not include the note at that time. 

Per Shadi's request, we will have another discussion in the 22 May meeting to enable him to participate in the discussion since he was unable to make the 16 May call.

**Comment by maryjom** [comment]
The TF made a [resolution to add a note to 3.3.8](https://www.w3.org/2024/05/30-wcag2ict-minutes#r08) in the 30 May meeting. The text is as follows:

>NOTE 4: There are cases where non-web software has an authentication process and no alternative or assistance mechanism is feasible, for example when entering a password when starting, powering on / turning on an ICT (device or otherwise). In such situations, it may not be possible for the non-web software to meet this success criterion.

**Comment by maryjom** [comment]
Closing this issue as complete, as Shadi's comment is also being tracked in the AG WG review issue #368. The changes will soon be merged once remaining edits on other topics have reached consensus and PR #378 is merged.
